### ISSUE TITLE
Lobby server's little bug

### ISSUE THREAD

**Issue by 7erm1nal** [Regalis11/Barotrauma#9457]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In lobby server, in creating new company, in signs "initial balance" and "initial stock" if I click '-', value is increases, and if '+' -- decreases

### Reproduction steps

1. Create a new server
2. Selerct "New campaign"

### Bug prevalence

Happens regularly

### Version

0.18.12.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This was in a sense intentional: the button on the right increases the difficulty and the one on the left decreases it. In the case of initial supplies and balance, increasing the difficulty means decreasing the value, which makes the plus/minus buttons misleading and confusing. Addressed in https://github.com/Regalis11/Barotrauma-development/commit/de4b9d07b4049860fb836cc89c0fc969717030d7 by changing the buttons to generic arrows.

**Comment by NilanthAnimosus** [comment]
Tested in 0.18.13.0

When I first started playing again - I tripped up over these values and was setting them the wrong direction I intended as its not intuitive, and when the left arrow for difficulty decreases it to Low over medium, I find it strange the left arrow changes starting balance from medium to high as that is an "Increase" in my brain, leading me to hit the other arrow I intended.

Although this is working fine as arrows over plus and minus buttons, I'm not sure why we are adding individually-different meaning on a value change based off what the value is, as the UI is in my opinion not immediately obvious as it should be as to what will happen by pressing them (left usually being decrease in most UI design even, I would expect a left to reduce the money I start with, not left for the easier option as this does seem unintuitive to me as a player).

Does it not make more sense to go with what most will find intuitive over a specific context that changes directions (in terms of less/more) to what the value is (difficulty/money/supplies)?
